kalite/distributed/static/js/distributed/topics/router.js

```diff
@@ -1,6 +1,8 @@
-ChannelRouter = Backbone.Router.extend({
+TopicChannelRouter = Backbone.Router.extend({
     initialize: function(options) {
+        _.bindAll(this);
         this.default_channel = options.default_channel;
+        $("#nav_learn").click(this.intercept_learn_nav);
```

Comment:
That seems like it should be a problem that's been solved by Backbone or its community? But maybe not.